fitstack/mcmc.py

```diff
@@ -21,6 +21,12 @@
 
 PERCENTILE = [2.5, 16, 50, 84, 97.5]
 
+SIMULATION_MODELS = [
+    "SimulationTemplate",
+    "SimulationTemplateFoG",
+    "SimulationTemplateFoGAltParam",
+]
+
```

Comment:
I think you can generate this dynamically which certainly would have saved me one exception when I tried to add a new model 


```suggestion
SIMULATION_MODELS = [
    c.__name__ for c in 
    [models.SimulationTemplate] + models.SimulationTemplate.__subclasses__()
]
```
or by filtering out the models module's globals dict I guess.